test-bom/build.gradle

```diff
@@ -3,15 +3,15 @@ plugins {
 }
 
 dependencies {
-    api platform("org.junit:junit-bom:$junitVersion")
-    api platform("org.spockframework:spock-bom:$spockVersion")
+    api(platform(libs.boms.junit))
```

Comment:
This is redundant, the `bom` plugin will automatically add this platform.

Reply:
Fixed.